content/200-concepts/100-components/01-prisma-schema/10-postgresql-extensions.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: 'PostgreSQL extensions'
+metaDescription: 'How to represent PostgreSQL extensions in your Prisma schema'
+hidePage: false
+preview: true
+tocDepth: 2
+---
+
+<TopBlock>
+
+This page introduces PostgreSQL extensions and describes how to represent extensions in your Prisma schema, how to introspect existing extensions in your database, and how to apply changes to your extensions to your database with Prisma Migrate.
+
+<Admonition type="warning">
+
+Support for declaring PostgreSQL extensions in your schema is available in preview for the PostgreSQL connector only in Prisma versions 4.5.0 and later.
+
+</Admonition>
+
+</TopBlock>
+
+## What are PostgreSQL extensions?
+
+PostgreSQL allows you to extend your database functionality by installing packages known as _extensions_. For example, the `citext` extension adds a case-insensitive string data type.
+
+To install an extension, it must already be available on the local file system of the your database server. Installing it then runs a script file that adds the new functionality.
+
+Prisma's `postgresExtensions` preview feature allows you to represent PostgreSQL extensions in your Prisma schema. Note that specific extensions may add functionality that is not currently supported by Prisma. For example, an extension may add a type or index that is not supported by Prisma. This functionality must be implemented on a case-by-case basis and is not provided by this preview feature.
+
+## How to enable the <inlinecode>postgresExtensions</inlinecode> preview feature
```

Comment:
Is `postgresExtensions` the correct name? I see it [here](https://www.notion.so/prismaio/PSL-Proposal-Postgres-Extensions-Declaration-177dbbc76b9340b2870f6c7a4641822a#a4d3b36912a04e62a7a1e5fb179e3ea5), but it feels a bit inconsistent as in the docs we always say PostgreSQL and not Postgres

Reply:
@pimeys is there any update on the name of the feature?

Reply:
Slack conversation here: https://prisma-company.slack.com/archives/C03SDE5G5L0/p1665568845383349